system-addon/test/unit/lib/ThemeFeed.test.js

```diff
@@ -0,0 +1,102 @@
+import {actionTypes as at} from "common/Actions.jsm";
+import {GlobalOverrider} from "test/unit/utils";
+import injector from "inject!lib/ThemeFeed.jsm";
+
+describe("ThemeFeed", () => {
+  let ThemeFeed;
+  let THEME_UPDATE_EVENT;
+  let feed;
+  let globals;
+  let sandbox;
+
+  beforeEach(() => {
+    globals = new GlobalOverrider();
+    sandbox = globals.sandbox;
+    ({ThemeFeed, THEME_UPDATE_EVENT} = injector({}));
```

Comment:
If nothing is being injected, why not just import the module normally?

Reply:
In case you asked me to make `LightweightThemeManager` global? 😉